site-modules/project/tasks/init.json

```diff
@@ -0,0 +1,22 @@
+// {
```

Comment:
Could we start them with a real task here? Maybe a simple "echo" or something.

Some of the features demonstrated in this sample metadata are a bit advanced, like `implementations`. Also, comments aren't supported here, which means this file will likely cause an error if you tried to run the task.